crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -17,10 +17,28 @@ use bevy_render::{
 #[bind_group_data(StandardMaterialKey)]
 #[uniform(0, StandardMaterialUniform)]
 pub struct StandardMaterial {
+    /// The color of the surface of the material before lighting.
+    ///
     /// Doubles as diffuse albedo for non-metallic, specular for metallic and a mix for everything
-    /// in between. If used together with a base_color_texture, this is factored into the final
+    /// in between. If used together with a `base_color_texture`, this is factored into the final
     /// base color as `base_color * base_color_texture_value`
+    ///
+    /// Defaults to [`Color::WHITE`].
     pub base_color: Color,
+
+    /// The texture component of the material's color before lighting,
+    /// the actual pre-lighting color is `base_color * this_texture`.
```

Comment:
```suggestion
    /// The texture component of the material's color before lighting.
    /// The actual pre-lighting color is `base_color * this_texture`.
```

Reply:
I realize now that the first sentence of struct fields is never displayed separately from the rest of the doc, so it is perfectly reasonable to split the sentence in two. I'll just ping @djeedai for the justification on why I decided to split the sentence, since I made this a single sentence on their request.

Reply:
You're sure? That doesn't ring any bell, and I agree with @superdump this should be 2 sentences.

Reply:
> That doesn't ring any bell,

Nice, I'm glad you approve, I must have misunderstood you.